Makefile

```diff
@@ -10,6 +10,8 @@ BUILDFLAGS := -tags "$(BUILDTAGS)"
 
 PACKAGES := $(shell go list ./... | grep -v github.com/containers/image/vendor)
 
+NO_GIT_VALIDATION = "registries.conf"
```

Comment:
Could this have a comment explaining why this is necessary?

Reply:
fixed